src/Draco.Compiler/Api/Syntax/SyntaxFactory.cs

```diff
@@ -104,10 +104,12 @@ public static SeparatedSyntaxList<TNode> SeparatedSyntaxList<TNode>(SyntaxToken
         Semicolon);
 
     public static FunctionDeclarationSyntax FunctionDeclaration(
+        SyntaxToken? visibility,
```

Comment:
Why not take Visibility here? Asking for a syntax token is really uncomfortable in terms of API